src/Features/VisualBasic/Portable/CodeLens/VisualBasicCodeLensMemberFinder.vb

```diff
@@ -0,0 +1,85 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Collections.Immutable
+Imports System.Composition
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.CodeLens
+Imports Microsoft.CodeAnalysis.Host.Mef
+Imports Microsoft.CodeAnalysis.PooledObjects
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.CodeLens
+    <ExportLanguageService(GetType(ICodeLensMemberFinder), LanguageNames.VisualBasic), [Shared]>
+    Friend NotInheritable Class VisualBasicCodeLensMemberFinder
+        Implements ICodeLensMemberFinder
```

Comment:
ok.  given that you just use the visitor pattern for each, i can see why you don't share any code.  that's fine with me.